sdk/communication/ci.yml

```diff
@@ -26,6 +26,8 @@ extends:
     ServiceDirectory: communication
     ArtifactName: packages
```

Comment:
Please include the directive ` BuildSnippets: true` to ensure that snippets are compiled.     [Example](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/ci.yml#L41)

Reply:
This has been added